engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/AncientDataMotionStrategy.java

```diff
@@ -146,7 +146,15 @@ private Scope pickCacheScopeForCopy(DataObject srcData, DataObject destData) {
     }
 
     protected Answer copyObject(DataObject srcData, DataObject destData, Host destHost) {
-        int primaryStorageDownloadWait = StorageManager.PRIMARY_STORAGE_DOWNLOAD_WAIT.value();
+        long dataSize = 0;
+        try{
+            dataSize = srcData.getSize();
+        }catch(NullPointerException e){
```

Comment:
```suggestion
        } catch(NullPointerException e) {
```

check and correct formatting in the code changes, wherever applicable.